products/bigquery/ansible_version_added.yaml

```diff
@@ -0,0 +1,207 @@
+---
```

Comment:
These files are incredibly verbose. Since they're meant for machines, I didn't see any reason to put in time to make them less repetitive. If users want to change them by-hand, they can.